impl/jamtemplate/common/game_state.cpp

```diff
@@ -63,7 +61,6 @@ void GameState::internalUpdate(float elapsed)
 
 void GameState::internalDraw() const
 {
-    getGame()->gfx().window().startRenderGui();
```

Comment:
unrelated to fixed update. Gui considerations should never have been part of `GameState` anyway.